ui/app/AppLayouts/Wallet/popups/simpleSend/SendSignModal.qml

```diff
@@ -256,6 +263,7 @@ SignTransactionModalBase {
     property Component internalPopup: TransactionSettings {
 
         fnGetPriceInCurrencyForFee: root.fnGetPriceInCurrencyForFee
+        fnGetPriceInNativeTokenForFee: root.fnGetPriceInNativeTokenForFee
         fnGetEstimatedTime: root.fnGetEstimatedTime
         fnRawToGas: function(rawValue) {
             return Utils.nativeTokenRawToGas(root.networkChainId, rawValue)
```

Comment:
This will return the BigJS number as string (e.g. `'45.6'`) so it needs to be formatted to the user's locale (potentially converting `.` into the correct decimal separator)

Reply:
For an example, see the `AmountToSend.qml` component how it handles the conversion when working with BigJS string-as-numbers

Reply:
I see what we have there, but why don't we have that somehow on the global level that I can call/reuse? 
What do you suggest?

Reply:
I think you're right! We could use a simple, reusable big to locale convertor. Converting app data to UI should be as simple as possible. And I can't find any 🤔 

@micieslak Do you know if we have a one stop localizer for Big numbers?


I'm thinking of something like this.
```
LocaleUtils.qml

function bigToLocaleStr(amount, decimals /* = default - is there one?*/) {
         let amount = amount.toFixed(decimals)
         amount = stripTrailingZeros(amount)
        return amount.replace(".", Qt.locale().decimalPoint)
}
```

Reply:
We have `LocaleUtils::currencyAmountToLocaleString` but I'm not sure if it's what you are looking for.

It would be handy to have some global stateless method for formatting amounts, but the problem is that formatting depends on symbol. For various symbols we can actually apply different formats. E.g. for AssetsViewAdaptor we pass following formatting function: https://github.com/status-im/status-desktop/blob/abd6fb871240ee86f275ea21188050377e3e2345/ui/app/AppLayouts/Wallet/views/RightTabView.qml#L233

So whenever we rely on store it's not proper solution to use global singleton there.